src/itemlistformaction.cpp

```diff
@@ -492,9 +492,13 @@ void ItemListFormAction::process_operation(Operation op,
 				    "markfeedread-jumps-to-next-unread")) {
 				process_operation(OP_NEXTUNREAD);
 			} else {
-				LOG(Level::DEBUG,
-					"ItemListFormAction::reset itempos");
-				f->set("itempos", "0");
+				std::string sortorder =
+					cfg->get_configvalue("article-sort-order");
+				if (sortorder.find("desc") == std::string::npos) {
+					LOG(Level::DEBUG,
+						"ItemListFormAction::reset itempos %s");
```

Comment:
The `%s` seem to be a leftover from a debugging session; can it be removed?

Reply:
Thanks for the feedback, I'll take another look tomorrow.